sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyClient.cs

```diff
@@ -479,15 +484,18 @@ public virtual Pageable<KeyProperties> GetPropertiesOfKeys(CancellationToken can
         }
 
         /// <summary>
-        /// Lists the properties of all keys in the specified vault. You can use the returned <see cref="KeyProperties.Name"/> in subsequent calls to <see cref="GetKeyAsync"/>.
+        /// Lists the properties of all enabled and disabled keys in the specified vault. You can use the returned <see cref="KeyProperties.Name"/> in subsequent calls to <see cref="GetKeyAsync"/>.
         /// </summary>
-        /// <remarks>
+        /// <para>
```

Comment:
Did you mean for the `<para>` tags to be contained by `<summary>`? 

In their current form, I don't believe this will render correctly for the MS Docs site;  the `<para>` tags are not in the `<summary>`  or `<remarks>` containers or any other identified section container.  When the tools walk the doc comments, they pull out specific nodes and process the contents, ignoring any other arbitrary top-level elements.  

For illustration, the basic structure looks like:
```csharp
return new Docs()
{
    Summary = NormalizeDocsElement(dElement.Element("summary"), out _),
    Remarks = (remarksAreFormatted) ? remarksText : FormatTextIntoParagraphs(remarksText),
    Examples = examplesText,
    AltMemberCommentIds = MergeAltmemberAndSeealsoToAltMemberCommentsIds(dElement),
    Related = related,
    Exceptions = dElement.Elements("exception")?.Select(el => GetTypedContent(el, filePath)).ToList(),
    Permissions = dElement.Elements("permission")?.Select(el => GetTypedContent(el, filePath)).ToList(),
    Parameters = dElement.Elements("param")?.Where(p => !string.IsNullOrEmpty(p.Attribute("name").Value)).ToDictionary(p => p.Attribute("name").Value, p => NormalizeDocsElement(p, out _)),
    TypeParameters = dElement.Elements("typeparam")?.Where(p => !string.IsNullOrEmpty(p.Attribute("name").Value)).ToDictionary(p => p.Attribute("name").Value, p => NormalizeDocsElement(GetInnerXml(p))),
    AdditionalNotes = additionalNotes,
    Returns = NormalizeDocsElement(dElement.Element("returns"), out _), 
    ThreadSafety = threadSafetyContent,
    ThreadSafetyInfo = threadSafety,
    Since = NormalizeDocsElement(dElement.Element("since")?.Value),
    AltCompliant = dElement.Element("altCompliant")?.Attribute("cref")?.Value,
    InternalOnly = dElement.Element("forInternalUseOnly") != null,
    Inheritdoc = inheritDoc
};
```

Reply:
Oh, I see what you mean. They are supposed to be in `<remarks>` like the file above. I'll fix.